filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -2455,10 +2454,10 @@ GLsizei OpenGLDriver::getAttachments(AttachmentArray& attachments,
         assert_invariant(!defaultFramebuffer);
         attachments[attachmentCount++] = GL_COLOR_ATTACHMENT7;
     }
-    if (any(buffers & TargetBufferFlags::DEPTH)) {
+    if (any(buffers & TargetBufferFlags::DEPTH) && rt->width != 0 && rt->height != 0) {
```

Comment:
Add a note to the PR desc. about this: 👴 I don't remember, ask @veghbalazs again please

Reply:
Added comment in the description.